services/event/service/event_service.go

```diff
@@ -176,6 +176,18 @@ func CreateEvent(id string, event models.Event) error {
 
 	err = db.Insert("eventtrackers", &event_tracker)
 
+	if err != nil {
+		return err
+	}
+
+	event_code := models.EventCode{
+		ID:         id,
+		Code:       code,
+		Expiration: 1521388800,
```

Comment:
What's this hardcoded value?

Reply:
This is an arbitrary time value prior to the event. It can be changed to be 0 honestly (I think I will do that actually), so the initial value is the start of epoch time.

Reply:
> This is an arbitrary time value prior to the event. It can be changed to be 0 honestly (I think I will do that actually), so the initial value is the start of epoch time.

That will make all of the codes expired by default? What if we do `event.endTime + SOME_GRACE_PERIOD`

Reply:
Expiration is now initialized to end time.